certified-connectors/Occuspace Occupancy Tracker/apiDefinition.swagger.json

```diff
@@ -0,0 +1,367 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "0.1",
+    "title": "Occuspace",
+    "description": "The Occuspace connector provides real-time and historical occupancy data for Occuspace-enabled locations. Show how many people are in your space in real-time and learn from past trends to let people know when to expect busy times.\n\nFor more information, contact us at info@occuspace.io.",
```

Comment:
The end of the description can be removed in favor of the contact information that you provided below. So, simply:

`"description": "The Occuspace connector provides real-time and historical occupancy data for Occuspace-enabled locations. Show how many people are in your space in real-time and learn from past trends to let people know when to expect busy times.`

Reply:
Fixed